src/main/java/com/ultimatelogger/multiplatform/SwitchableUltimateLogger.kt

```diff
@@ -0,0 +1,6 @@
+package com.ultimatelogger.multiplatform
+
+interface SwitchableUltimateLogger : UltimateLogger {
```

Comment:
Type `com.ultimatelogger.multiplatform.` in an app that uses JvmLog or the android version and see what IDE suggests, because you may not want all those classes and interfaces to be user-visible.

Reply:
It will be complicated, tasked.